webapp/src/components/MediaSource.vue

```diff
@@ -171,7 +171,7 @@ export default {
 			// Set the language iframe URL when language changes
 			this.languageIframeUrl = this.getLanguageIframeUrl(languageUrl)
 		},
-		getYoutubeUrl(ytid, autoplay, mute, hideControls, noRelated, showinfo, disableKb, loop, modestBranding, enablePrivacyEnhancedMode) {
+		getYoutubeUrl (ytid, autoplay, mute, hideControls, noRelated, showinfo, disableKb, loop, modestBranding, enablePrivacyEnhancedMode) {
```

Comment:
Why space between function name and `(`?

Reply:
it could be because after  I run lint with the configuration "Prettier" ?